ci-operator/config/openshift-assisted/assisted-installer-deployment/openshift-assisted-assisted-installer-deployment-master.yaml

```diff
@@ -0,0 +1,34 @@
+base_images:
+  coreos_centos_stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.build
+  inputs:
+    coreos_centos_stream8:
+      as:
+      - quay.io/centos/centos:stream8
+  to: assisted-installer-deployment-build
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pycodestyle
```

Comment:
Can we call it with a more generic name like `lint`?

Reply:
done